### ISSUE TITLE
FakeWifi v1.3 and android 10 with edxposed

### ISSUE THREAD

**Issue by nnrs1** [chylek/FakeWifi#19]
Hi, on my S10+, FakeWifi v1.3 works like a charm with android pie and edxposed, thx.
Yesterday i installed "android 10"on my Samsung S10+. My news attempts are unsuccessful works with FakeWifi v1.3, edxposed and "PS4 Remote Play". The error message is " A connection via a wi-fi network cannot be established. Chek your network connection.
Any ideas?
Best regards
Sorry for my bad english, i'm french.

**Comment by LeeBinder** [comment]
@nnrs1 @chylek  the latest version I see everywhere is v.1.2. Can any of you (or someone else) please attach v.1.3 apk, or post a link? Thank you.

**Comment by chylek** [comment]
@LeeBinder Yeah, I didn't test it, so I didn't publish it on Google Play or anywhere else. Download directly from github: https://github.com/chylek/FakeWifi/releases/download/v1.3/FakeWifi.v1.3.apk 
There's only slim chance it will work though. I don't have use for it anymore and there are far too many changes in latest Android versions.

**Comment by LeeBinder** [comment]
@chylek great to hear from you, and thanks for the link. v1.3 indeed works on my Nokia 6.1 in LineageOS 18.1 with Android 11 + Magisk Zygisk LSPosed. But still Google Play Store complains about no Internet in conjunction with gnirehtet USB reverse tethering in both, macOS and Windows. Same with lemonsqueeze's original Fake Wifi Connection v.1.8/ 1.10 Beta, and same when using Florian Draschbacher's ReverseTethering NoRoot instead gnirehtet. Installing the latest Google Connectivity Service app did not help, either.

I'm aware this is a commonly known issue _for some_ but not for all. On a Nokia 6.2, USB reverse tethering was working fine for me - but that one had the stock Android 10 ROM on it.

During my research I've not found a solution like some script to install via Magisk to fix Google Play disregarding Internet via USB reverse tethering .. :(

**Comment by prudhvinadhreddy06** [comment]
@chylek @LeeBinder 

Device:Moto G 5G, Android 11
Installed Magisk and LSPosed.
Tried these apps but no luck
https://github.com/lemonsqueeze/FakeWifiConnection

https://github.com/chylek/FakeWifi/releases/download/v1.3/FakeWifi.v1.3.apk

I am using "gnirehtet" for reverse USB tethering.

After opening the FakeWifi app i am getting "can't find xposed" error. But other xposed modules are working fine.

Any workaround/help is appreciated.

Or share resources for creating a new module for this purpose(I had some experience in development)

**Comment by LeeBinder** [comment]
@prudhvinadhreddy06 maybe [this post by evolu1](https://github.com/Genymobile/gnirehtet/issues/397#issuecomment-872851962) helps you. But I need to admit that evolu1's approach did not work for me with all apps. So after successfully running the free 3-day trial of Tetrd ([tetrd.app](https://tetrd.app/) | [Google Play](https://play.google.com/store/apps/details?id=com.robskie.tether) | [XDA](https://forum.xda-developers.com/t/app-4-1-tetrd-usb-tethering-reverse-tethering-noroot.4032777/)), I happily purchased the pro-expansion via its in-App purchase for €6,49 and never regretted that small investment or looked back.

For Tetrd, please do in any case first read [Tetrd - best sequence](https://forum.xda-developers.com/t/app-4-1-tetrd-usb-tethering-reverse-tethering-noroot.4032777/post-88242981).
